lib/OpenQA/Jobs/Constants.pm

```diff
@@ -78,3 +78,10 @@ our @EXPORT = qw(
   MODULE_RESULTS
   TIMEOUT_EXCEEDED
 );
+
+my %GENERALIZED_RESULTS = ((map { $_ => $_ } OK_RESULTS, FAILED), (map { $_ => 'incomplete' } INCOMPLETE_RESULTS));
```

Comment:
uh, care to put a comment explain this thing?

Reply:
added a comment